src/navigator/DrawerNavigator.tsx

```diff
@@ -16,6 +16,7 @@ import {
   useLinkBuilder,
 } from '@react-navigation/native'
 import { TransitionPresets } from '@react-navigation/stack'
+import { Home } from 'branding/valora/src/icons/Home'
```

Comment:
Think this import should be `src/icons/Home`

Reply:
Yup, I did it. Thank you!